deploy/scripts/src/mobile/backend-e2e.test.ts

```diff
@@ -0,0 +1,60 @@
+import { type Options } from 'k6/options'
+import { describeProfile, type ProfileList, selectProfile } from '../common/utils/config/load-profiles'
+import {
+  postVerifyAuthorizeRequest,
+  postResourceOwnerDocumentGroups,
+  getBiometricTokenV2,
+  postFinishBiometricSession,
+  getRedirect,
+  postToken,
+  postUserInfoV2
+} from './utils/backend'
+import { sleep } from 'k6'
+
+const profiles: ProfileList = {
+  smoke: {
+    backendJourney: {
+      executor: 'ramping-arrival-rate',
+      startRate: 1,
+      timeUnit: '1s',
+      preAllocatedVUs: 50,
+      maxVUs: 70,
+      stages: [
+        { target: 5, duration: '30s' },
+        { target: 5, duration: '30s' }
+      ],
+      exec: 'backendJourney'
+    }
+  }
+}
+
+const loadProfile = selectProfile(profiles)
+
+export const options: Options = {
+  // httpDebug: 'full',
+  scenarios: loadProfile.scenarios,
+  thresholds: {
+    http_req_duration: ['p(95)<=1000', 'p(99)<=2500'], // 95th percentile response time <=1000ms, 99th percentile response time <=2500ms
```

Comment:
Is this something people feel strongly about including? We kept it as it was left over from the FE journey and seems like reasonable expectations of API calls. All it seems to do is add a ✅  or ❌  next to the test output if it fails, and logs a message if the threshold isn't met. 

The POST /userinfo/v2 call frequently causes the 95th percentile to exceed 1000ms anyway, so not sure what value this really adds other than flagging that. 
<img width="1048" alt="image" src="https://github.com/alphagov/di-devplatform-performance/assets/109979772/96eb1e39-7e9a-4a6d-8056-91fd61e65109">

Reply:
I feel strongly about it! It's automatic threshold validation that will cause the test to fail if criteria aren't met.

It checks these two NFRs @jamielutton-gds 

<img width="790" alt="image" src="https://github.com/alphagov/di-devplatform-performance/assets/115020902/2f3c44bc-cd3c-440d-b637-8acade99a8ac">

Reply:
I'm not surprised we see it failing when running smoke tests - we'll hit lambda cold starts for both userInfo and the readId mock.

Reply:
Thanks for the context! What does this mean then - do our NFRs have to change to account for this? Or in the future, will we have finer-grained tests, that would check (as an example) something like p(95) < 1s for all endpoints except userinfo, but p(95)<1.5 for userinfo only?

Reply:
We'll need to run this performance test at load to get a full picture of how userInfo really performs

Reply:
These NFRs are set at a programme level and if we can't meet them we have a defect to triage/fix

Reply:
That makes sense. Thanks!